ecl/hql/hqlgram2.cpp

```diff
@@ -11773,6 +11790,8 @@ extern HQL_API IHqlExpression * parseQuery(IHqlScope *scope, IFileContents * con
                 StringBuffer s;
                 ctx.errs->reportError(ERR_INTERNALEXCEPTION, E->errorMessage(s).str(), str(sourcePath), 0, 0, 1);
             }
+            else if (E->errorCode() == ERR_ABORT_PARSING)
```

Comment:
This is not the previous behaviour.  Previously if this occurred, the exception would not have been thrown, now it will be.  It will result in various places receiving internal exceptions.